traffic_monitor/health/cache.go

```diff
@@ -114,11 +123,7 @@ func GetVitals(newResult *cache.Result, prevResult *cache.Result, mc *tc.Traffic
 			MaxKbpsOut: iface.Speed * 1000,
 		}
 
-		if prevResult != nil && prevResult.InterfaceVitals != nil && prevResult.InterfaceVitals[ifaceName].BytesOut != 0 {
-			elapsedTimeInSecs := float64(newResult.Time.UnixMilli()-prevResult.Time.UnixMilli()) / 1000
-			if elapsedTimeInSecs == 0 {
-				elapsedTimeInSecs = float64(prevResult.Time.UnixMilli() / 1000)
-			}
+		if prevResult.InterfaceVitals != nil && prevResult.InterfaceVitals[ifaceName].BytesOut != 0 {
```

Comment:
`prevResult` can still be `nil` here, so I think we should keep that check in here.